site/en/articles/webcodecs/index.md

```diff
@@ -207,7 +207,7 @@ while (true) {
     frame.close();
   } else {
     frameCounter++;
-    const keyframe = frameCounter % 150 == 0;
+    const keyFrame = frameCounter % 150 == 0;
     encoder.encode(frame, { keyFrame });
```

Comment:
`encoder.encode(frame, { keyFrame })` uses `keyFrame` whose "F" is an upper case.

Reply:
[Confirmed.](https://developer.mozilla.org/en-US/docs/Web/API/VideoEncoder/encode#syntax)